docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,267 @@
+        Console.WriteLine(json.GetDateTime()); // 7/26/2019 12:00:00 AM
+    }
+}
+```
+## Custom support for date and times in `JsonSerializer`
+
+If you want custom parsing or formatting at the serializer level, you can implement [custom converters](https://github.com/dotnet/corefx/issues/36639). Here are a few examples:
+
+### Using `DateTime(Offset).Parse` and `DateTime(Offset).ToString`.
+
+This allows you to use .NET's extensive support for parsing various date and time formats, including non-ISO 8601 strings and ISO 8601 formats that don't conform to the Extended ISO 8601-1:2019 profile.
+This approach can be used if you can't determine the input date and time formats, but it is significantly less performant than using the serializer's native implementation.
+
+[!code-csharp[example-showing-datetime-parse](~/samples/snippets/standard/datetime/json/datetime-converter-examples/example1/Program.cs)]
+
+### Using `UTF8Parser` and `UTF8Formatter`.
+
+This allows you to use fast parsing and formatting methods for UTF-8 date and time data that is compliant with one of the [Standard Date and Time Format Strings](https://docs.microsoft.com/dotnet/standard/base-types/standard-date-and-time-format-strings).
+This is much faster than Example 1 and should be used if the input data isn't compliant with the Extended ISO 8601-1:2019 profile but conforms to the "R", "l", "O", or "G" standard format specifiers.
+
+[!code-csharp[example-showing-utf8-parser-and-formatter](~/samples/snippets/standard/datetime/json/datetime-converter-examples/example2/Program.cs)]
+
+### Using `DateTime(Offset).Parse` as a fallback to the serializers native parsing.
+
+This approach can be used if you generally expect the input date and time data to conform to the Extended ISO 8601-1:2019 profile, but want to have a fallback just in case.
+
+```csharp
+public class DateTimeConverterExample3 : JsonConverter<DateTime>
+{
+    public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        if (reader.TokenType != JsonTokenType.String)
+        {
+            throw new JsonException();
+        }
+
+        ReadOnlySpan<byte> valueSpan = reader.ValueSpan;
+
+        if (!reader.TryGetDateTime(out DateTime value))
+        {
+            value = DateTime.Parse(Encoding.UTF8.GetString(valueSpan));
+        }
+
+        return value;
+    }
+
+    public override void Write(Utf8JsonWriter writer, DateTime value, JsonSerializerOptions options)
+    {
+        writer.WriteStringValue(value.ToString());
+    }
+}
+```
+
+## Extended ISO 8601-1:2019 Profile in System.Text.Json
+
+### Date and time components
+
+ISO 8601-1:2019 defines the following components for date and time representations:
+
+| Component         | Format                                                     | ISO 8601-1:2019 spec | Description                                                                                                     |
```

Comment:
This table contains helpful info but I wonder if we can sanitize it in some way to avoid spec-specific terms and maybe use terms people who read the dotnet docs might be more familiar with. Like what's done here:
https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings